cypress/fixtures/graphql.js

```diff
@@ -76,6 +76,7 @@ export const mocks = {
   // Custom scalars:
   URL: () => 'https://www.example.com/',
   DateTime: () => new Date().toISOString(),
+  Date: () => new Date().toISOString(),
```

Comment:
@mendelB does this make sense? I figured it would be similar but not sure 🤔

Reply:
I think this is fine! If we want to match the [expected format](https://github.com/DoSomething/graphql/blob/80c0bce1a5f4bdcef8f06313d0481c92f87a311c/src/schema/northstar.js#L109) (`YYYY-MM-DD`) perfectly, we can just append a `slice` to grab only this part of the string I think.

```suggestion
  Date: () => new Date().toISOString().slice(0, 10),
```

`# '2021-01-12T16:42:47.230Z' -> '2021-01-12'`